### ISSUE TITLE
migrate/implement the registration page in react as a component

### ISSUE THREAD

**Issue by kwlskiy** [threedevs/3dev-fullstack#23]
only build the component no validation or request needed yet

**Comment by asubedy** [comment]
Hey I like to work on this. So basically I have to design the signup page from the given figma design page link given in the wireframe prototype?
https://www.figma.com/file/XIUfEV7DlJHJCxwN8BAz5U/Library?node-id=1%3A3

**Comment by kwlskiy** [comment]
yes, you can assign yourself the task and make the react component for it, if you feel comfortable you can also add the states and the handlers for the events.

**Comment by chawlapalak** [comment]
Hi @kwlskiy  I would like to work on this.

**Comment by chawlapalak** [comment]
hi can i work on this
